src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -113,6 +115,10 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
           if (data.url.toLowerCase().includes('oauth/authorize')) {
             this.updateOauthSession(this.signInLocal.params)
           } else {
+            this._gtag.reportEvent('RegGrowth', 'Sign-In', 'Website').subscribe(
+              () => (this.window.location.href = data.url),
+              () => (this.window.location.href = data.url)
+            )
             this.navigateTo(data.url)
```

Comment:
So I think we won't require this navigate anymore, right?

Reply:
Yes, It is not required anymore. I will remove it.